src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    inversedRobot[robot[key]] = key;
+  }
+
+  const robotLength = Object.entries(robot).length;
+  const inversedRobotLength = Object.entries(inversedRobot).length;
+
+  if (robotLength !== inversedRobotLength) {
+    return null;
+  }
+
+  return inversedRobot;
```

Comment:
if should check if new keu exists in loop
```suggestion
  for (const key in robot) {
    const newKey = robot[key];
    
    if (inversedRobot[newKey]) {
      return null;
    }
    
    inversedRobot[newKey] = key;
  }

  return inversedRobot;
```